test/lib/content-function.js

```diff
@@ -49,8 +49,15 @@ export function testContentFunctions(t, message, fn) {
             thumb,
             to,
             urls,
+
             appendIndexHTML: false,
+
             getColors: c => getColors(c, {chroma}),
+            getDimensionsOfImagePath: () => [600, 600],
+            getThumbnailEqualOrSmaller: () => 'medium',
+            getThumbnailsAvailableForDimensions: () =>
+              [['large', 800], ['medium', 400], ['small', 250]],
+
```

Comment:
These here should probably just be cut and pasted into the `quickSnapshot` function in `test/snapshot/image.js`, since that's already the only place in test cases defining `getSizeOfImagePath` (née `getSizeOfImageFile`).